presidio-anonymizer/presidio_anonymizer/operators/aes_cipher.py

```diff
@@ -1,8 +1,8 @@
 import base64
+import os
 
-from Crypto import Random
-from Crypto.Cipher import AES
-from Crypto.Util.Padding import pad, unpad
+from cryptography.hazmat.primitives import padding
```

Comment:
Please update NOTICE too (remove pycryptodome's OSS license and add cryptography's)